### DISCUSSION TITLE
RITA data no longer ingested in SO 2.3.140?

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#8352] [Unsupported Versions] [answered]
Hello!
Did you drop the ingest of RITA data in So 2.3.140? It's no longer working and I can't find anything about the ingest support in the current documentation. :-(
Cheers, Ben

**Comment by weslambert** [comment]
I just tried this on a fresh 2.3.140 install, and it worked fine for me. 

If you updated from 2.3.130, you might need to run `so-rita-restart`.

Are there any clues in `/opt/so/log/filebeat/filbeat*.log` or `/opt/so/log/logstash/logstash.log`?

**Reply by ben-sec** [reply]
I found the following error in one of the filebeat logs that is pointing to rita-connectiosn:
```
{"log.level":"error","@timestamp":"2022-07-22T14:12:59.473Z","log.logger":"input.filestream","log.origin":{"file.name":"filestream/prospector.go","file.line":294},"message":"Error while stopping harverster group: task failures\n\terr    or while adding new reader to the bookkeeper harvester is already running for file\n\terror while adding new reader to the bookkeeper harvester is already running for file","service.name":"filebeat","id":"rita-connection","prospector    ":"file_prospector","ecs.version":"1.6.0"}
```

**Comment by 0nnyx** [comment]
Using Wes rita docker installation, it works fine - https://github.com/weslambert/securityonion-rita
Hope one day to get the html report available on nginx

**Reply by ben-sec** [reply]
rita is running fine and puts the necessary files in /nsm/rita, but the ingest is not working.

**Reply by 0nnyx** [reply]
I had a worse problem on 2.3.120 when using rita and mongo installed manually as it even froze ingestion of other data. So if you have the minion properly configured for ingestion (rita:  enabled: True), you might try Wes docker which works fine on 2.3.140

**Comment by ben-sec** [comment, marked as answer]
My Elasticsearch node reached the max number of shards again and stopped to index data (first RITA, then Suricata, ...). Suggestion: Maybe you want to display the current number of shards in Grafana? However, I will now setup a second search node to avoid this issue in the future.
